tests/t406-qfunction.h

```diff
@@ -6,11 +6,15 @@
 // This file is part of CEED:  http://github.com/ceed
 
 // Note: intentionally testing strange spacing in '#include's
```

Comment:
Could probably also do similar testing for `#pragma once` as well, since you're trying to allow spaces between `#` and `pragma`.

Reply:
Updated